coops-ui/src/views/AnnualReport.vue

```diff
@@ -163,6 +163,8 @@
                 :certifiedBy.sync="certifiedBy"
                 :currentDate="currentDate"
                 @valid="certifyFormValid=$event"
+                :entityDisplay="displayName()"
+                :message="certifyText('OTANN')"
```

Comment:
Maybe another spot we could use the filingCodes enum?

Reply:
In this case however, yes it does make sense. Good catch!